site/blog/OIS_Sessions_Blog

```diff
@@ -1,29 +1,23 @@
 ---
-title: StarlingX sessions from Open Infrastructure Summit
+title: The Cure for StarlingX FOMO at Open Infrastructure Summit
```

Comment:
I liked the previous title better without the acronym. As this goes out to an international audience it is better to keep the title easier to understand.

Reply:
I had just realized the previous title was pretty boring. i edited the intro to define the term right away. Hows that?

Reply:
I'm not the biggest fan, but let's get this post landed so we can get the content out. Thanks for all the fixes!